docs/pbc/all/search/202212.0/search.md

```diff
@@ -8,7 +8,30 @@ redirect_from:
   - /docs/scos/dev/feature-walkthroughs/202212.0/search-feature-walkthrough.html
 ---
 
-Search is one of the most effective navigation tools in modern shops. The Search capability lets you create the right search for your shop and enable customers to quickly find what they are looking for.
+Search is one of the most effective navigation tools in modern shops. The Search capability lets you create the right search for your shop and enables customers to quickly find what they are looking for.
+
```

Comment:
TAs this page will be a landing page for the search PBC, we should make it more meaningful. Let's expand this page with information from [this one](https://docs.spryker.com/docs/scos/user/features/202212.0/search-feature-overview/search-types-overview.html) and redirect from that page here. Also, the schema in the presentation (slide 5) is very good - let's use it her as well and describe the components shown in the schema.

Reply:
It says 'Page Not Found' on that link @lenadoc